soyoooooon/Northern_shopify
# products reviews

### Goal
Add Product review app to store

### Deployment Instructions
- Pull to master 
- Ensure that your production theme id is correct in 'config.yml'
-`./theme deploy --env production`

### Testing Instructions
- Go to the PDP first
- Go to the [collection page](https://l8slupfy0p8g4fzn-31233605770.shopifypreview.com)
- Go down and write review
- Scroll to the top of the page, ensure that the reviews badge showing up near the price


### Tested On
Chrome on Mac OS

### Code Request Quality Checklist


### Reviewers
@reviewer1  
@reviewer2  

### Mentions
@mention1  
@mention2
